src/main/java/frc/robot/maps/FrancoisMap.java

```diff
@@ -0,0 +1,101 @@
+package frc.robot.maps;
+
+import com.chopshop166.chopshoplib.RobotMapFor;
+import com.chopshop166.chopshoplib.maps.DifferentialDriveMap;
+import com.chopshop166.chopshoplib.outputs.EncodedSpeedController;
+import com.chopshop166.chopshoplib.outputs.PIDSparkMax;
+import com.chopshop166.chopshoplib.outputs.SendableSpeedController;
+import com.chopshop166.chopshoplib.outputs.WDSolenoid;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.revrobotics.CANEncoder;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.revrobotics.EncoderType;
+
+@RobotMapFor("Francois")
+public class FrancoisMap extends RobotMap {
+
+    @Override
+    public DifferentialDriveMap getDriveMap() {
+        return new DifferentialDriveMap() {
+
+            @Override
+            public EncodedSpeedController getRight() {
+                CANSparkMax leader = new CANSparkMax(20, MotorType.kBrushless);
+                CANSparkMax follower = new CANSparkMax(21, MotorType.kBrushless);
+                CANEncoder leadEncoder = new CANEncoder(leader, EncoderType.kQuadrature, 42);
+                follower.follow(leader);
+
+                final double distancePerPulse = (1.0 / 46.0) * (1.0 / 12.27) * (6.0 * Math.PI);
```

Comment:
This should be defined up a level so it can be shared between the left and right.

Reply:
Finished